fullscreen.bs

```diff
@@ -1,25 +1,12 @@
 <pre class=metadata>
-Title: Fullscreen API Standard
 Group: WHATWG
 H1: Fullscreen API
 Shortname: fullscreen
-Status: LS
-No Editor: true
+Text Macro: TWITTER fullscreenapi
 Abstract: The Fullscreen API standard defines an API for elements to display themselves fullscreen.
-Logo: https://resources.whatwg.org/logo-fullscreen.svg
-Boilerplate: omit feedback-header, omit conformance, omit index, omit idl-index
-!Participate: <a href=https://github.com/whatwg/fullscreen>GitHub whatwg/fullscreen</a> (<a href=https://github.com/whatwg/fullscreen/issues/new>file an issue</a>, <a href=https://github.com/whatwg/fullscreen/issues>open issues</a>)
-!Participate: <a href=https://wiki.whatwg.org/wiki/IRC>IRC: #whatwg on Freenode</a>
-!Commits: <a href=https://github.com/whatwg/fullscreen/commits>GitHub whatwg/fullscreen/commits</a>
-!Commits: [SNAPSHOT-LINK]
-!Commits: <a href=https://twitter.com/fullscreenapi>@fullscreenapi</a>
-!Tests: <a href=https://github.com/w3c/web-platform-tests/tree/master/fullscreen>web-platform-tests fullscreen/</a> (<a href=https://github.com/w3c/web-platform-tests/labels/fullscreen>ongoing work</a>)
+Translation: ja https://triple-underscore.github.io/fullscreen-ja.html
```

Comment:
Hmm, should we do this in a separate commit and also acknowledge @triple-underscore? That's what I proposed for the Notifications API.

Reply:
Sure, I'll remove it from here.